scalameta/trees/shared/src/main/scala/scala/meta/internal/ast/InternalTrees.scala

```diff
@@ -44,6 +44,8 @@ trait InternalTree {
     if (privateParent != null) scala.Some(privateParent) else None
   }
 
+  def productFields: Seq[String]
```

Comment:
Let's keep it. I'd like to say `extends Product` here just for the sake of documentation.